src/Symfony/Component/Notifier/Bridge/LineBot/LineBotTransportFactory.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Notifier\Bridge\LineBot;
+
+use Symfony\Component\Notifier\Exception\UnsupportedSchemeException;
+use Symfony\Component\Notifier\Transport\AbstractTransportFactory;
+use Symfony\Component\Notifier\Transport\Dsn;
+
+/**
+ * @author Yi-Jyun Pan <me@pan93.com>
+ */
+final class LineBotTransportFactory extends AbstractTransportFactory
+{
+    private const SCHEME = 'linebot';
+
+    protected function getSupportedSchemes(): array
+    {
+        return [self::SCHEME];
+    }
+
+    public function create(Dsn $dsn): LineBotTransport
+    {
+        if (self::SCHEME !== $dsn->getScheme()) {
+            throw new UnsupportedSchemeException($dsn, self::SCHEME, $this->getSupportedSchemes());
+        }
+
+        $accessToken = $dsn->getUser();
+        if (null === $accessToken) {
+            throw new \InvalidArgumentException('You must provide a channel access token as "user" in the DSN.');
+        }
```

Comment:
```suggestion
        $accessToken = $this->getUser();

```

Reply:
![CleanShot 2024-10-11 at 07 57 37@2x](https://github.com/user-attachments/assets/644c917f-5ae0-4d46-8589-a1579373d951)

![CleanShot 2024-10-11 at 07 57 18@2x](https://github.com/user-attachments/assets/3e806977-80ca-477a-9b3b-1631a3a3971e)

It might be needed. It may throw a confused error if users do not provide an access token.

Reply:
Oh. I change to `IncompleteDsnException` now.